public/keymaps/a/annepro2_c15_default.json

```
@@ -0,0 +1,29 @@
+{
+  "keyboard": "annepro2/c15",
+  "keymap": "default",
+  "version": 1,
+  "layout": "LAYOUT_60_ansi",
+  "layers": [
+    [
+      "KC_GESC", "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS",        "KC_EQL",         "KC_BSPC",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC",        "KC_RBRC",        "KC_BSLS",
+      "LT(1,KC_CAPS)",      "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN",        "KC_QUOT",        "KC_ENT",
+      "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH",                          "RSFT_T(KC_UP)",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "LT(1,KC_LEFT)",  "LT(2,KC_DOWN)",  "RCTL_T(KC_RGHT)"
+    ],
+    [
+      "KC_GRV",  "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",  "KC_DEL",
+      "_______", "_______", "KC_UP",   "_______", "_______", "_______", "_______", "_______", "_______", "_______", "KC_PSCR", "KC_HOME", "KC_END", "_______",
+      "_______", "KC_LEFT", "KC_DOWN", "KC_RGHT", "_______", "_______", "_______", "_______", "_______", "_______", "KC_PGUP", "KC_PGDN",           "_______",
+      "_______",            "KC_VOLU", "KC_VOLD", "KC_MUTE", "_______", "_______", "_______", "_______", "_______", "KC_INS",  "KC_DEL",            "_______",
+      "_______", "_______", "_______",                                  "_______",                                  "_______", "_______", "MO(2)", "_______"
+    ],
+    [
+      "_______",  "ANY(KC_AP2_BT1)",   "ANY(KC_AP2_BT2)",   "ANY(KC_AP2_BT3)",   "ANY(KC_AP2_BT4)",   "_______",   "_______",   "_______",   "_______",   "RGB_MOD",   "RGB_TOG",  "RGB_VAI",  "RGB_VAD",  "_______",
+      "MO(2)"   , "_______",           "KC_UP",             "_______",           "_______",           "_______",   "_______",   "_______",   "_______",   "_______",   "KC_PSCR",  "KC_HOME",  "KC_END",   "_______",
+      "_______",  "KC_LEFT",           "KC_DOWN",           "KC_RGHT",           "_______",           "_______",   "_______",   "_______",   "_______",   "_______",   "KC_PGUP",  "KC_PGDN",              "_______",
+      "_______",                       "_______",           "_______",           "_______",           "_______",   "_______",   "_______",   "_______",   "_______",   "KC_INS",   "KC_DEL",               "_______",
+      "_______",  "_______",           "_______",                                  "_______",                                                                          "_______",  "MO(1)",    "MO(2)",    "_______"
```

The `KC_AP2_*` keycodes don't need to be pre-assigned as `ANY()` keycodes - Configurator's code handles that automatically. Also, the `RGB_*` keycodes didn't match their `qmk/qmk_firmware` assignments.

```suggestion
      "_______", "KC_AP2_BT1", "KC_AP2_BT2", "KC_AP2_BT3", "KC_AP2_BT4", "_______", "_______", "RGB_HUI", "RGB_SAI", "RGB_VAI", "RGB_SPI", "RGB_MOD", "RGB_TOG", "_______",
      "MO(2)",   "_______",    "KC_UP",      "_______",    "_______",    "_______", "_______", "_______", "_______", "_______", "KC_PSCR", "KC_HOME", "KC_END",  "_______",
      "_______", "KC_LEFT",    "KC_DOWN",    "KC_RGHT",    "_______",    "_______", "_______", "_______", "_______", "_______", "KC_PGUP", "KC_PGDN",            "_______",
      "_______",               "_______",    "_______",    "_______",    "_______", "_______", "_______", "_______", "_______", "KC_INS",  "KC_DEL",             "_______",
      "_______", "_______",    "_______",                                           "_______",                                  "_______", "MO(1)",   "MO(2)",   "_______"
```